PKI authn in Kibana (#485)
elastic/stack-docs: docs/en/stack/security/authentication/pki-realm.asciidoc

(PKI) certificates to authenticate users in {es}. This requires clients to
present X.509 certificates.

You can use PKI certificates to authenticate users in {es} as well as {kib}.

To use PKI in {es}, you configure a PKI realm, enable client authentication on
the desired network layers (transport or http), and map the Distinguished Names

> (DNs) from the user certificates to roles in the <<pki-role-mapping, role

Reviewer: ```suggestion
(DNs) from the user certificates to roles. You create the mappings in a <<pki-role-mapping, role
```

> mapping file or using the API>>. If you wish that the same users also be

Reviewer: ```suggestion
mapping file>> or use the {ref}/security-api-put-role-mapping.html[create role mappings API]. If you want the same users to also be
```

> authenticated using certificates when they connect to {kib}, then the {es} PKI

Reviewer: ```suggestion
authenticated using certificates when they connect to {kib}, the {es} PKI
```

> realm needs to be configured to

Reviewer: ```suggestion
realm to
```
